src/app/calculator/steam/stack-loss/stack-loss-help/stack-loss-help.component.html

```diff
@@ -228,7 +228,7 @@ <h6>
       Moisture in Air
       <br>
       <small class="text-muted">
-        The moisture of the air (mass water / mass dry air). Can be determined with a psychrometric chart.
+        The percent moisture of the air (mass water / mass dry air). Can be determined with a psychrometric chart.
```

Comment:
Since the moisture in air combustion field has been removed from the UI of the Stack Loss calc, this help text will never be seen by the users, so it needs to be moved to a section of the help text that is shown no matter what field is selected.